### PR TITLE
Support the case=False argument to str.contains

### PR DESCRIPTION
**Pull request by shwina**
Closes https://github.com/rapidsai/cudf/issues/13253

### CODE REVIEW

File: python/cudf/cudf/core/column/string.py [hunk 1]
```diff
@@ -767,22 +765,31 @@ def contains(
...
+
+        if is_scalar(pat):
             if regex:
                 result_col = libstrings.contains_re(self._column, pat, flags)
             else:
-                result_col = libstrings.contains(
-                    self._column, cudf.Scalar(pat, "str")
-                )
+                if case is False:
+                    input_column = libstrings.to_lower(self._column)
```
Comment (mroeschke) [inline python/cudf/cudf/core/column/string.py:778]: Nit: Guessing libstrings doesn't have a `casefold`-like method right? https://docs.python.org/3/library/stdtypes.html#str.casefold

Comment (shwina) [reply python/cudf/cudf/core/column/string.py:778]: @davidwendt is (or was this ever) on our radar?

Comment (davidwendt) [reply python/cudf/cudf/core/column/string.py:778]: No.

**Review by mroeschke — APPROVED** [review]
LGTM

**Comment by vyasr** [comment]
Is this PR waiting on anything?

**Pull request merged**